using_images/s2i_images/java.adoc

```diff
@@ -69,43 +82,43 @@ By default, the Java S2I builder image uses Maven to build the project with the
 following goals and options:
 
 ----
-mvn -Dmaven.repo.local=/tmp/artifacts/m2 -s /tmp/artifacts/configuration/settings.xml -e -Popenshift -DskipTests -Dcom.redhat.xpaas.repo.redhatga -Dfabric8.skip=true package -Djava.net.preferIPv4Stack=true
+mvn -e -Popenshift -DskipTests -Dcom.redhat.xpaas.repo.redhatga -Dfabric8.skip=true --batch-mode -Djava.net.preferIPv4Stack=true -s /tmp/artifacts/configuration/settings.xml -Dmaven.repo.local=/tmp/artifacts/m2  package
 ----
 
 Based on these defaults, the builder image compiles the project and copies all
 the transitive dependencies into the output directory without running tests.
-Additionally, if the project has a profile named `*openshift*`, then it is
+Additionally, if the project has a profile named `openshift`, it is
 activated for the build.
 
 You can override these default goals and options by specifying the following environment variables:
 
-.Java Environment Variables
-[options="header"]
 |===
-
 |Variable name |Description
 
-|`*ARTIFACT_DIR*`
-|The relative path to the target where JAR files are created for multi-module builds.
+|`*MAVEN_S2I_ARTIFACT_DIRS*`
```

Comment:
Maybe update the variable formatting from `* to just ` throughout.

Reply:
@bmcelvee Just so I am clear, should I remove the bold formatting from all the environment variable names?